### PR TITLE
working team+chat reset tests

### PR DESCRIPTION
**Pull request by maxtaco**
- !!! it works !!!
- fix all bugs required to get it to work
- Fix a subtle but in UPAK2 to v1 conversion -- sometimes we need error + data out of load (cc: @oconnor663)
- get rid of ugly debugging

### CODE REVIEW

File: go/libkb/upak_loader.go [hunk 1]
```diff
@@ -377,11 +380,16 @@ func (u *CachedUPAKLoader) loadWithInfo(arg LoadUserArg, info *CachedUserLoadInf
 // backwards compatibility.
 func (u *CachedUPAKLoader) Load(arg LoadUserArg) (*keybase1.UserPlusAllKeys, *User, error) {
 	ret, user, err := u.loadWithInfo(arg, nil, nil, true)
-	if err != nil {
-		return nil, nil, err
+
+	// NOTE -- it's OK to return an error and a user, since certain code paths
+	// want both (see note in loadWithInfo).
```
Comment (maxtaco) [inline go/libkb/upak_loader.go]: @oconnor663 check out the above...

**Comment by keybase-ci-visdiff** [comment]
The commits d17351dd7b6d...22973469c21c introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Tracker-New_user_Scroll_1** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1498434347240-d17351dd7b6d-22973469c21c-linux/Tracker-New_user_Scroll_1-d17351dd7b6d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1498434347240-d17351dd7b6d-22973469c21c-linux/Tracker-New_user_Scroll_1-22973469c21c.png)  
   Tracker-New_user_Scroll_1 comparison
</summary>

File: go/libkb/upak_loader.go (hunk 1 above)
Comment (oconnor663) [inline go/libkb/upak_loader.go]: Good to know!

File: go/client/cmd_team_list_memberships.go [hunk 2]
```diff
@@ -54,24 +76,24 @@ func (c *CmdTeamListMemberships) Run() error {
 		return err
 	}
 
-	members, err := cli.TeamGet(context.Background(), keybase1.TeamGetArg{Name: c.team})
+	details, err := cli.TeamGet(context.Background(), keybase1.TeamGetArg{Name: c.team, ForceRepoll: true})
```
Comment (patrickxb) [inline go/client/cmd_team_list_memberships.go]: Shouldn't this use c.forcePoll?

File: go/systests/multiuser_common_test.go [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+	service "github.com/keybase/client/go/service"
+	clockwork "github.com/keybase/clockwork"
+	rpc "github.com/keybase/go-framed-msgpack-rpc/rpc"
+	contextOld "golang.org/x/net/context"
+	"io"
+	"testing"
+	"time"
+)
+
+type smuUser struct {
```
Comment (patrickxb) [inline go/systests/multiuser_common_test.go]: What does `smu` stand for?

**Comment by maxtaco** [comment]
Needs a comment sorrry about that.  Systest Multi User.

**Comment by maxtaco** [comment]
absolutely. thanks for the review

will work on cleaning it up

**Comment by maxtaco** [comment]
CI failures unrelated 😭 😭

**Pull request merged**